odc/stats/model.py

```diff
@@ -318,15 +318,8 @@ def _lineage(self) -> Tuple[UUID, ...]:
 
         # TODO: replace this and test
         # if 'fused' in ds.metadata._doc['properties'].keys():
-        if "fused" in ds.type.name:
-            lineage = tuple(
-                set(
-                    x
-                    for ds in self.datasets
-                    for y in ds.metadata.sources.values()
-                    for x in y.values()
-                )
-            )
+        if "fused" in ds.product.name:
+            lineage = tuple(set(x for ds in self.datasets for x in ds.metadata.sources))
```

Comment:
Not sure if I implemented this wrong in the first place or if I'm misremembering now - doesn't the new `ds.metadata.sources` still have the classifiers as the keys? I think this needs to use `ds.metadata.sources.values()`

Reply:
Good to know my past self knew what she was doing!